packages/frontend/src/lib/pnl.ts

```diff
@@ -113,6 +113,38 @@ const getRelevantSwaps = (squeethAmount: BigNumber, swaps: swaps_swaps[], isWeth
   return relevantSwaps
 }
 
+const getSwapsWithEthPrice = async (swaps: swaps_swaps[]) => {
+  const timestamps = swaps.map((s) => Number(s.timestamp) * 1000)
+
+  // Don't fetch if already fetched
+  const ethPriceMap = await queryClient.fetchQuery([timestamps], () => getHistoricEthPrices(timestamps), {
```

Comment:
We could remove the square brackets on the query key since `timestamps` is an array.

Reply:
My bad will change it